lib/utils/export/to-zip.js

```diff
@@ -0,0 +1,21 @@
+import JSZip from 'jszip';
+
+export const noteExportToZip = notes => {
+	const zip = new JSZip();
+
+	notes
+		.activeNotes
+		.forEach(
+			( { id, content } ) => zip.file( `${ id }.txt`, content )
```

Comment:
> If there's a way that we can use the note's first line, that'd be cool.

how should we handle duplicates and blank note titles?

for duplicates I could imagine `document.txt` and `document (1).txt` but for blank notes?

 - grab the first N-characters of content -> `third line of note maybe.txt`
 - `untitled.txt` and `untitled (1).txt`